openedx/core/djangoapps/content/course_metadata/utils.py

```diff
@@ -0,0 +1,22 @@
+"""
+Utility methods for course metadata app
+"""
+from django.conf import settings
+
+from xmodule.modulestore.django import modulestore
+
+
+def get_course_leaf_nodes(course_key):
```

Comment:
@mattdrayer added `test_utils.py` module with tests for operations in `utils.py`.